service.kn.switchtimer/addtimer.py

```diff
@@ -0,0 +1,17 @@
+import xbmc
+import handler
+
+if __name__ ==  '__main__':
+
+    handler.notifyLog('Parameter handler called: add Timer')
+    try:
+        args = {'channel':xbmc.getInfoLabel('ListItem.ChannelName'),
```

Comment:
yes you might be right, the PVRInfoTag is not available via python afaik.
Ignore this then.